packages/pf-codemods/index.js

```diff
@@ -43,8 +44,15 @@ function printResults(engine, results, format) {
 
 function main() {
   let currentOptions;
+
   try {
-    currentOptions = options.parse(process.argv);
+    let arguments = lodash.cloneDeep(process.argv);
```

Comment:
I think `[...process.argv]` works? No need for lodash

Reply:
I was trying to be safe with using lodash.deepcopy(), but I am sure it is over kill for this. I have changed back to the above suggestion.